zipline/src/commonMain/kotlin/app/cash/zipline/ZiplineApiMismatchException.kt

```diff
@@ -1,3 +1,18 @@
+/*
+ * Copyright (C) 2022 Block, Inc.
```

Comment:
I didn't see any announcement on this, we're good to switch? should we? is it optional?

Reply:
Yes. Use `Block, Inc.` for new files. Don‘t retroactively fix anything existing.